sdk/schemaregistry/azure-schemaregistry-avroserializer/azure/schemaregistry/serializer/avroserializer/_version.py

```diff
@@ -24,4 +24,4 @@
 #
 # --------------------------------------------------------------------------
 
-VERSION = "1.0.0b2"
+VERSION = "1.0.0b3"
```

Comment:
could you log an issue to bump the azure-schemaregistry dependency in setup.py and shared_requirements.txt + re-record tests after azure-schemaregistry b3 is released?